### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Fixes misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/vscode-pull-request-github/actions/runs/6803799840#summary-18499918007

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/vscode-pull-request-github/actions/runs/6803799902#summary-18499918517

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ about: Create a report to help us improve
 - VSCode Version:
 - OS:
 - Repository Clone Configuration (single repository/fork of an upstream repository):
-- Github Product (Github.com/Github Enterprise version x.x.x):
+- GitHub Product (GitHub.com/GitHub Enterprise version x.x.x):
```
Comment (jsoref) [inline .github/ISSUE_TEMPLATE/bug_report.md:14]: GitHub is a brand.

Note that I didn't change GitHubFoo class things as I couldn't decide if they were public APIs that people would care about.

File: CHANGELOG.md [hunk 2]
```diff
@@ -165,7 +165,7 @@
 - My vscode workspace sometimes shows a PR from vscode-cpptools. https://github.com/microsoft/vscode-pull-request-github/issues/4842
 - Improper `@mentions` in comments. https://github.com/microsoft/vscode-pull-request-github/issues/4810
 - Duplicated issues in tree. https://github.com/microsoft/vscode-pull-request-github/issues/4781
-- Element with id Local Pull Request Brancheshttps... is already registered. https://github.com/microsoft/vscode-pull-request-github/issues/4642
+- Element with id Local Pull Request Branches https... is already registered. https://github.com/microsoft/vscode-pull-request-github/issues/4642
```
Comment (jsoref) [inline CHANGELOG.md:168]: some projects don't like changing changelogs...

File: CHANGELOG.md [hunk 3]
```diff
@@ -201,7 +201,7 @@
 
 * [@Balastrong (Leonardo Montini)](https://github.com/Balastrong)
   * Add x button to remove a label from a new PR [PR #4649](https://github.com/microsoft/vscode-pull-request-github/pull/4649)
-  * Change file mode for execute husky hook on MacOS [PR #4695](https://github.com/microsoft/vscode-pull-request-github/pull/4695)
+  * Change file mode for execute husky hook on macOS [PR #4695](https://github.com/microsoft/vscode-pull-request-github/pull/4695)
```
Comment (jsoref) [inline CHANGELOG.md:204]: brand...

File: package.nls.json [hunk 4]
```diff
@@ -174,8 +174,8 @@
 	"command.pr.refreshChanges.title": "Refresh",
 	"command.pr.configurePRViewlet.title": "Configure...",
 	"command.pr.deleteLocalBranch.title": "Delete Local Branch",
-	"command.pr.signin.title": "Sign in to GitHub",
-	"command.pr.signinenterprise.title": "Sign in to GitHub Enterprise",
+	"command.pr.signin.title": "Sign into GitHub",
+	"command.pr.signinenterprise.title": "Sign into GitHub Enterprise",
```
Comment (jsoref) [inline package.nls.json:178]: There's a difference of opinion about this... I personally favor `into` but some argue that `sign-in` is some sort of magical verb that doesn't hyphenate (verbs don't...) but justifies `Sign in` + `to GitHub` -- which doesn't really make sense to me as I don't `GitHub` as a verb. I treat GitHub as a destination that I can go _into_.

- https://github.com/community/community/discussions/68753#discussion-5685029
https://code.visualstudio.com/docs/sourcecontrol/github
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/converting-a-user-into-an-organization

-- that said, I'm happy to drop this or anything else.

File: src/common/utils.ts [hunk 5]
```diff
@@ -123,7 +123,7 @@ export function isDescendant(parent: string, descendant: string): boolean {
 		parent += sep;
 	}
 
-	// Windows is case insensitive
+	// Windows is case-insensitive
```
Comment (jsoref) [inline src/common/utils.ts:126]: beyond the grammar this isn't technically correct. It depends on the file system and the configuration of the file system (or even folder? https://www.windowscentral.com/how-enable-ntfs-treat-folders-case-sensitive-windows-10 )

File: src/github/graphql.ts [hunk 6]
```diff
@@ -615,7 +615,7 @@ export interface PullRequestResponse {
 	rateLimit: RateLimit;
 }
 
-export interface PullRequestMergabilityResponse {
+export interface PullRequestMergeabilityResponse {
```
Comment (jsoref) [inline src/github/graphql.ts:618]: While I didn't change GitHub* APIs, I've changed this one...

**Comment by jsoref** [comment]
@microsoft-github-policy-service agree

**Comment by alexr00** [comment]
Thank you for the PR, but we don't usually take PRs that only fix spelling errors (similar to https://github.com/microsoft/vscode/wiki/How-to-Contribute#spell-check-errors)

**Pull request closed without merging**